dali/kernels/imgproc/resample/resampling_batch.cu

```diff
@@ -92,7 +87,7 @@ __global__ void BatchedSeparableResampleKernel(
     }
     break;
   case ResamplingFilterType::Linear:
-    if (axis == 1) {
+    if (axis == spatial_ndim - 1) {
       LinearHorz(x0, x1, y0, y1, origin, scale, sample_out, out_stride, sample_in, in_stride,
```

Comment:
Now this Horz/Vert naming is a bit misleading.

Reply:
idea: you could have some constants to make this more readable:
(e.g `kHorizontalAxis = 0` and `kHorizontalDim = spatial_ndim - 1`)